app/src/main/res/layout/fragment_second.xml

```diff
@@ -9,27 +9,94 @@
 
     <!-- TODO: Update blank fragment layout -->
 
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="413dp"
+        android:layout_height="339dp"
+        android:layout_marginStart="4dp"
```

Comment:
You need to create a dimension for the spaces of very items on the screen. That way is consistent across the app.